ui/device.ui

```diff
@@ -92,49 +143,170 @@ background-color: green;
 }
 QPushButton[failed=true] {
 background-color: red;
+}*/</string>
+        </property>
+        <property name="text">
+         <string/>
+        </property>
+        <property name="icon">
+         <iconset resource="../resources/resources.qrc">
+          <normaloff>:/menu/ic adalm.png</normaloff>:/menu/ic adalm.png</iconset>
+        </property>
+        <property name="iconSize">
+         <size>
+          <width>100</width>
+          <height>100</height>
+         </size>
+        </property>
+        <property name="checkable">
+         <bool>true</bool>
+        </property>
+        <property name="flat">
+         <bool>false</bool>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QLabel" name="description">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
+        <property name="styleSheet">
+         <string notr="true">QLabel {
+color: white;
+font-family: &quot;monospace&quot;;
```

Comment:
You should remove the font because it overrides the font set in the app

Reply:
Sure, sorry. Should be fixed with commit bc08231.